wavelength-plugins/misc.js

```diff
@@ -402,53 +402,55 @@ exports.commands = {
 			return WL.messageSeniorStaff(msg);
 		case 'color':
 			if (target[1].substring(0, 1) !== '#' || target[1].length !== 7) return this.errorReply(`Colors must be a 6 digit hex code starting with # such as #009900`);
-			msg += '/html <center>' + WL.nameColor(user.name, true) + ' has redeemed a custom color token.<br/> hex color: <span' + target[1] + '<br/>';
-			msg += '<button class="button" name="send" value="/customcolor set ' + user.name + ',' + target[1] + '">Set color (<b><font color="' + target[1] + '">' + target[1] + '</font></b>)</button></center>';
+			msg += `/html <center>${WL.nameColor(user.name, true)} has redeemed a custom color token.<br/> Hex color: <span ${target[1]}<br/>`;
```

Comment:
It looks like this span tag has been messed up for a while, can you remove it?

Reply:
Sure